### DISCUSSION TITLE
Sort delegates by structure level for elections

### DISCUSSION THREAD

**Discussion by lwbr** [OpenSlides/OpenSlides#5986] [Ideas]
In the case of elections in which the voting rights of participants change for example they leave a board and are then too old or cannot get a replacement vote, it would be cool for admins to have an extra overview page where delegates are sorted by structure level. This assumes that the structure level, similar to the groups, can be created beforehand and then assigned to the participant. On the overview page you can see a list of all the by structure levels with the total number of suffrages and you can open it up to see the participants and edit them.

**Comment by tsiegleauq** [comment]
I think we'll pick that up in 4.1 (or so)

https://github.com/OpenSlides/openslides-client/issues/722

Structure level will be handles as a model like tags.

**Comment by jsangmeister** [comment]
As a participant can now have multiple structure levels, this is no longer really feasible, so I'm closing this
